tmail-backend/mailets/src/main/scala/com/linagora/tmail/mailets/ContactsCollection.scala

```diff
@@ -52,11 +52,20 @@ class ContactsCollection @Inject()(eventBus: EventBus) extends GenericMailet {
         contact = contact),
         NO_REGISTRATION_KEYS)))
       .collectSeq()
-      .subscribeOn(Schedulers.elastic())
       .block()
 
   private def appendAttributeToMail(mail: Mail, contacts: Seq[ContactFields]): Unit =
     mail.setAttribute(new Attribute(attributeName,
-      AttributeValue.of(contacts.map(_.address.toString).mkString("[", ",", "]"))))
+      AttributeValue.of(ContactFieldSerializer.serialize(contacts).toString())))
```

Comment:
No please don't change the format here.

Reply:
I thought you wanted personal info?

Reply:
We want it in our index.

However the attribute value is used to pass the collected contact to the calendar and we need the format of what we send to the calendar to be stable.

If not we will have a bug (non handled message means existing collected contact feature broken) and possibly an outage (I don't trust them managing failed consumers nicely).

Reply:
I got it